libs/form-fields/src/lib/user-list-field.component.ts

```diff
@@ -365,9 +371,16 @@ export class UserListFieldComponent
     public addUsersFromFile(event) {
         /* istanbul ignore else */
         if (event.target) {
-            const file = event.target.files[0];
+            const file: File = event.target.files[0];
             /* istanbul ignore else */
             if (file) {
+                if (
+                    !ACCEPTED_FILE_TYPES.includes(file.type) ||
+                    DENIED_FILE_TYPES.includes(file.type)
+                ) {
+                    notifyError('File type not supported');
```

Comment:
Can we improve this error to say `File type not supported: Please upload a CSV file`